site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -1412,7 +1412,8 @@ public function ajaxSaveComponent($gradeable_id) {
         $default = $_POST['default'] ?? null;
         $max_value = $_POST['max_value'] ?? null;
         $upper_clamp = $_POST['upper_clamp'] ?? null;
-        $peer = $_POST['peer'] ?? 'false';
+        // You cannot change the "peer-ness" of a component.
+        //$peer = $_POST['peer'] ?? 'false';
```

Comment:
let's remove the commented out code here. I'm not necessarily against the comment, but it should probably be moved into the comment for the function, rather than in the body of the function.